src/pages/taskQueue/index.tsx

```diff
@@ -30,11 +31,11 @@ const TaskQueue = () => {
   const navigate = useNavigate();
 
   const [selectedDistro, setSelectedDistro] = useState(null);
-
+  usePageTitle(`Task Queue - ${distro}`);
   const { data: distrosData } = useQuery<
     TaskQueueDistrosQuery,
     TaskQueueDistrosQueryVariables
-  >(TASK_QUEUE_DISTROS, { fetchPolicy: "cache-and-network" });
```

Comment:
I think Arjun added the fetch policy so that the page would reload when a user navigates to it, is there a reason you removed it?

Reply:
Thanks I missed that!